README.md

```diff
@@ -47,6 +47,16 @@ subdirectories.  A comprehensive [ciao cluster setup
 document](https://clearlinux.org/documentation/ciao-cluster-setup.html)
 is also available.
 
+If you would like to contribute to ciao, check our [Contributing
+guide](https://github.com/01org/ciao/blob/master/CONTRIBUTING.md).
+The [wiki](https://github.com/01org/ciao/wiki/Single-Machine-Development-Environment)
+page ilustrates how to easily setup a development environment without
+needing a cluster and we recommend to take a look a the ['janitorial'
```

Comment:
I'd split this sentence into two and reword slightly

...without needing a cluster.  We also recommend taking a look at the janitorial bugs in our list of open issues as these bugs can be solved without an extensive knowledge of ciao.'